server/polar/event/endpoints.py

```diff
@@ -138,6 +146,100 @@ async def list(
     )
 
 
+@router.get(
+    "/hierarchy-stats",
+    summary="Get Hierarchy Statistics",
+    openapi_extra={"parameters": [get_metadata_query_openapi_schema()]},
+)
```

Comment:
Until we're 100% sure of the API format _and_ that it's useful to officially expose, I would advocate to make it a private endpoint (i.e. excluded from documentation and SDK). To do so add the argument: `    tags=[APITag.private]`